templates/addPublication.html

```diff
@@ -0,0 +1,29 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <h2>Enter Publication Details</h2>
+    <form action="/addpublication" method="post">
+        <label for="Pname">Paper Name:</label>
+        <input type="text" id="Pname" name="Pname" value="Scalable Machine"><br><br>
+        <label for="Cname">Conference name:</label>
+        <input type="text" id="Cname" name="Cname" value="ICMLA"><br><br>
+        <label for="Cyear">Conference Year:</label>
+        <input type="text" id="Cyear" name="Cyear" value="Doe"><br><br>
+        <label for="Status">Status:</label>
+        <input type="text" id="Status" name="Status" value=""><br><br>
+        <label for="Team">Team (Shortform):</label>
+        <input type="text" id="Team" name="Team" value="MG"><br><br>
+        <label for="Authors">Authors(Separate each one by ;):</label>
+        <input type="text" id="Authors" name="Authors" value="Naveen;Nanda"><br><br>

```

Comment:
Some changes here:
* I think it's better to have lowercase `id`s and `name`s.
* Avoid default values.
* Use dropdown with options for status, the only possible values are Accepted and Published (check the file, I'm not too sure).
* Teams should also be a dropdown.
* Authors separated by semicolon doesn't seem ideal. Is there a way we can get all the people at SF and have a multi-select here? Possibly handled through the Python code if required.